### PR TITLE
Panorama docs revamp and bug fix

### PR DESCRIPTION
**Pull request by roysagi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29460
fixes: https://github.com/demisto/etc/issues/28829
fixes: https://github.com/demisto/etc/issues/30226

## Description
Adding new README to Panorama.
Fixing an issue where get-url-from-host command failed

### CODE REVIEW

File: Packs/PAN-OS/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS",
     "description": "Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.",
     "support": "xsoar",
-    "currentVersion": "1.6.3",
+    "currentVersion": "1.6.4",
```
Comment (bakatzir) [inline Packs/PAN-OS/pack_metadata.json:5]: can be a major

File: Packs/PAN-OS/ReleaseNotes/1_6_4.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Documentation and metadata improvements.
+- Fixed an issue where the ***panorama-get-url-category-from-host*** command failed when fetching a URL.
```
Comment (bakatzir) [inline Packs/PAN-OS/ReleaseNotes/1_6_4.md:5]:
```suggestion
- Fixed an issue where the ***panorama-get-url-category-from-host*** command failed when querying a URL.
```

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 3]
```diff
@@ -1,7681 +1,4323 @@
...
-<p>&nbsp;</p>
-<h5>Command Example</h5>
-<p>
-  <code>!panorama-get-wildfire-configuration template=WildFire</code>
-</p>
-<h5>Context Example</h5>
-<pre>
+This integration supports both Palo Alto Networks Panorama and Palo Alto Networks Firewall. You can create separate instances of each integration, and they are not necessarily related or dependent on one another.
+Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.
+This integration was integrated and tested with version 8.1.0 of Palo Alto Firewall, Palo Alto Panorama
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/README.md:3]:
```suggestion
This integration was integrated and tested with version 8.1.0 and 9.0.1 of Palo Alto Firewall, Palo Alto Panorama
```

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 4]
```diff
@@ -1,7681 +1,4323 @@
...
+
+## Panorama Playbook
+* **PanoramaCommitConfiguration** : Based on the playbook input, the Playbook will commit the configuration to Palo Alto Firewall, or push the configuration from Panorama to predefined device groups of firewalls. The integration is available from Demisto v3.0, but playbook uses the GenericPooling sub-playbook, which is only available from Demisto v4.0.
+* **(Deprecated) PanoramaQueryTrafficLogs** : Use the Panorama Query Logs playbook instead.W raps the following commands with genericPolling to enable a complete flow to query traffic logs.
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/README.md:7]: this is the new `Panorama Query Logs`, so can remove.

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 5]
```diff
@@ -1,7681 +1,4323 @@
...
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| server | Server URL \(e.g., https://192.168.0.1\) | True |
+| port | Port \(e.g 443\) | False |
+| key | API Key | True |
+| device_group | Device group - Panorama instances only \(write shared for Shared location\) | False |
+| vsys | Vsys - Firewall instances only | False |
+| template | Template - Panorama instances only. | False |
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/README.md:69]:
```suggestion
| template | Template - Panorama instances only | False |
```

**Review by bakatzir — COMMENTED** [review]
looks good.
@kirbles19 WDYT to add `"useCases"` to the pack_emetadata? any suggestions ?

File: Packs/PAN-OS/pack_metadata.json (hunk 1 above)
Comment (roysagi) [reply Packs/PAN-OS/pack_metadata.json:5]: Why?, its only a bug fix and documentation that is not represented in XSOAR itself

Comment (bakatzir) [reply Packs/PAN-OS/pack_metadata.json:5]: true.

**Review by bakatzir — APPROVED** [review]

File: Packs/PAN-OS/Integrations/Panorama/README.md (hunk 4 above)
Comment (roysagi) [reply Packs/PAN-OS/Integrations/Panorama/README.md:7]: @bakatzir - your are talking about PanoramaQueryTrafficLogs specifically right?

**Comment by roysagi** [comment]
@kirbles19 - If you can please check the new panorama docs here.

**Comment by roysagi** [comment]
@kirbles19 - to make your check easier here are the commands that I added docs for, the rest existed in the [previous doc](https://github.com/demisto/content/blob/master//Packs/PAN-OS/Integrations/Panorama/README.md)
* panorama-get-predefined-threats-list
* panorama-block-vulnerability
* url
*  panorama-get-url-category-from-cloud
* panorama-get-url-category-from-host
* panorama-register-user-tag
* panorama-unregister-user-tag
* panorama-security-policy-match
* panorama-list-static-routes
* panorama-get-static-route
* panorama-add-static-route
* panorama-delete-static-route
* panorama-show-device-version
* panorama-download-latest-content-update
* panorama-content-update-download-status
* panorama-install-latest-content-update
* panorama-content-update-install-status
* panorama-check-latest-panos-software
* panorama-download-panos-version
* panorama-download-panos-status
* panorama-install-panos-version
* panorama-install-panos-status
* panorama-device-reboot
* panorama-show-location-ip
* panorama-get-licenses

**Comment by ShirleyDenkberg** [comment]
@bakatzir Doc review completed.  Fixed a few spelling mistakes in ContextPath names. 
Fixed spelling of Category
- contextPath: Panorama.Spyware.Rules.Cateogry
- contextPath: Panorama.Vulnerability.Rules.Cateogry
- contextPath: Panorama.Spyware.Rule.Cateogry
- contextPath: Panorama.Vulnerability.Rule.Cateogry

Fixed spelling of Extension
contextPath: File.Extenstion

This may cause backwards compatibility issues, but was but Bar told me to make the changes.

**Pull request merged**